File: 09-hash-table/main.py
from HashTable import HashTable

table = HashTable()
table.print()
table.set("Sinar", "sinar@gmail.com")
table.set("Elvis", "elvis@gmail.com")
table.set("Tane", "tane@gmail.com")
table.print()
print("[get] " + table.get("Sinar"))
table.set("Gerti", "gerti@gmail.com")
table.set("Arist", "arist@gmail.com")
table.print()
print("[get] " + table.get("Sinar"))

File: 09-hash-table/main.cs
using System;
using metwally;

class Program
{
    public static void Main(string[] args)
    {

      HashTable<string, string> table = new HashTable<string, string>();
      table.Print();
      table.Set("Sinar", "sinar@gmail.com");
      table.Set("Elvis", "elvis@gmail.com");
      table.Set("Tane", "tane@gmail.com");
      table.Print();
      Console.WriteLine("[get] " + table.Get("Sinar"));
    //Console.WriteLine("[get] " + table.Get("Tane"));
      table.Set("Gerti", "gerti@gmail.com");
      table.Set("Arist", "arist@gmail.com");
      table.Print();
      Console.WriteLine("[get] " + table.Get("Sinar"));
  
    }
}
